samples/XAMLCatalog/XAMLCatalog/UIKitControls/UIButtonWithControlsViewController.m

```diff
@@ -0,0 +1,87 @@
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "UIButtonWithControlsViewController.h"
+#import <UIKit/UIButton.h>
+
+@implementation UIButtonWithControlsViewController {
+    MenuTableViewController* _menuTVC;
+}
+
+- (void)viewDidLoad {
```

Comment:
>viewDidLoad [](start = 8, length = 11)

nit; would be nice to have comments to describe the config of test view controller itself.